apps/cookbook/src/app/examples/radio-example/examples/in-item.ts

```diff
@@ -0,0 +1,35 @@
+import { Component } from '@angular/core';
+
+import { stringifyPretty } from '~/app/shared/code-viewer/code-viewer.component';
+
+const items = [
+  { title: 'Bacon', value: 1 },
+  { title: 'Salami (disabled)', value: 2, disabled: true },
+  { title: 'Tenderloin', value: 3 },
+];
+
+const config = {
+  selector: 'cookbook-radio-in-item-example',
+  template: `<kirby-card>
+  <kirby-radio-group [value]="selected">
+    <kirby-item *ngFor="let item of items">
+      <kirby-radio [value]="item" slot="start" [disabled]="item.disabled"></kirby-radio>
+      <kirby-label>{{item.title}}</kirby-label>
```

Comment:
Consider just using the "text" for radio button styled in context of the item, instead of having multiple ways of implementation using kirby-label? (maybe I'm just misunderstanding the usage of the "text" prop is a label or used as hint/error message)

Reply:
This is intentional - when used within an item, the radio and it's label goes into different slots (consistent with Ionics implementation). This makes it possible to right align the radio for example.